src/test/java/Homework18Test.java

```diff
@@ -0,0 +1,47 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.time.Duration;
+
+public class Homework18Test extends BaseTest  {
+
+    @Test
+    public void playSong() throws InterruptedException {
+
+        //STEP 2: Login using valid credentials
+        WebElement emailField = driver.findElement(By.cssSelector("input[placeholder='Email Address']"));
+        emailField.clear();
+        emailField.sendKeys("elliott.nance@testpro.io");
+        Thread.sleep(2000);
+
+        WebElement passwordField = driver.findElement(By.cssSelector("input[type='password']"));
+        passwordField.clear();
+        passwordField.sendKeys("HondaAccord2024$");
+        Thread.sleep(2000);
+
+        WebElement submitBtn = driver.findElement(By.cssSelector("[type='submit']"));
+        submitBtn.click();
+        Thread.sleep(2000);
+
+
+        //STEP 3: Click Play next song button.
+        WebElement clickPlayNextSongBtn = driver.findElement(By.cssSelector("[title='Play next song']"));
+        clickPlayNextSongBtn.click();
+        Thread.sleep(2000);
+
+        //STEP 4: Click play button, to play a song
+        WebElement clickPlayBtn = driver.findElement(By.cssSelector("[title='Play or resume']"));
+        clickPlayBtn.click();
+        Thread.sleep(2000);
+
+        //STEP 5: Validate song is playing.
+        WebElement songPlaying = driver.findElement(By.cssSelector("div[data-test='soundbars']"));
+        Thread.sleep(2000);
```

Comment:
**Use in test methods, the following logical split:   GIVEN – WHEN - THEN**
    
For example

```
 @Test

    public void testChangeProfileName (){
        // GIVEN

        String newName  = getRandomString();

        String currentPassword = "te$t$tudent";

        provideEmail("demo@class.com");

        providePassword();

        clickSubmitBtn();

 

        // WHEN

        openUserProfilePage();

        setName(newName);

        setCurrentPassword(currentPassword);

 

 

        // THEN

        Assert.assertTrue(getSuccessPopUp().isDisplayed());

}
```

Please, add Assert

**One test - One Assert**

For example

![18](https://github.com/Test-Pro-Bootcamp/A57-QA-Automation-Framework/assets/104761482/16575851-6486-40d3-b467-5ad1dcf5e668)